debezium-core/src/main/java/io/debezium/config/Field.java

```diff
@@ -1175,4 +1175,13 @@ public static int isDouble(Configuration config, Field field, ValidationOutput p
         return 0;
     }
 
-}
\ No newline at end of file
+    public static int isNotNull(Configuration config, Field field, Field.ValidationOutput problems) {
```

Comment:
It'd be nice to have JavaDoc for this method. Granted, the others don't yet so understand that you're just following the existing code. But in this case, it'd be useful to know how this method differs from `isRequired`.